index.d.ts

```diff
@@ -139,12 +140,32 @@ interface RemoveOptions {
   dispatchAction: boolean
 }
 
+/**
+ * https://firebase.google.com/docs/reference/js/firebase.database
+ */
+
+interface FirebaseDatabaseService {
+  database: {
+    (app?: string): DatabaseTypes.FirebaseDatabase
+    Reference: DatabaseTypes.Reference
+    Query: DatabaseTypes.Query
+    DataSnapshot: DatabaseTypes.DataSnapshot
+    enableLogging: typeof DatabaseTypes.enableLogging
+    ServerValue: DatabaseTypes.ServerValue
+    Database: typeof DatabaseTypes.FirebaseDatabase
+  }
+}
+
 /**
  * Firestore instance extended with methods which dispatch
  * redux actions.
  * @see https://react-redux-firebase.com/docs/api/firebaseInstance.html
  */
-interface ExtendedFirebaseInstance extends DatabaseTypes.FirebaseDatabase {
+interface BaseExtendedFirebaseInstance
```

Comment:
Really like that you combined this all into one it much more clearly matches the name